### ISSUE TITLE
Suggestion: Systemd unit files

### ISSUE THREAD

**Issue by dodexahedron** [bvanassche/scst#29]
Since a lot of distributions are using systemd, at this point, it would be nice if any build on a system using systemd would install a systemd unit file instead of an init script.
Sure systemd generates one when you enable the service, but it of course isn't optimal and, on at least one of my systems, loads in a bad order without manually modifying the generated unit, resulting in the need to restart scst after boot, or else not all modules are loaded or the configuration is "invalid" due to missing mount points, and it's thus basically non-functional until a `systemctl restart scst`.

This is especially true if, for example, using fileio while also using zfs, since zfs might not load before scst without modifying that unit.

That's a specific example, but I'm sure a more generic case can be set up that would avoid the majority of such situations, like specifying `After=multi-user.target` or something like that.

**Comment by Tualua** [comment]
ALT Linux has unit files for scst.

[scst-systemd.zip](https://github.com/bvanassche/scst/files/11238379/scst-systemd.zip)

**Comment by dodexahedron** [comment]
Yeah I made my own for all my CentOS and Ubuntu systems running SCST.
The same unit files work for all of them, and are just based on a trimmed-down version of what systemd generated automatically from the sysv-init service that scst currently installs.
